packages/signature-v4-multi-region/src/SignatureV4MultiRegion.ts

```diff
@@ -82,28 +76,52 @@ export class SignatureV4MultiRegion implements RequestPresigner, RequestSigner {
     return this.sigv4Signer.presignWithCredentials(originalRequest, credentials, options);
   }
 
-  private getSigv4aSigner(): InstanceType<OptionalCrtSignerV4> {
+  private getSigv4aSigner(): InstanceType<OptionalCrtSignerV4> | InstanceType<OptionalSigV4aSigner> {
     if (!this.sigv4aSigner) {
       let CrtSignerV4: OptionalCrtSignerV4 | null = null;
+      let JsSigV4aSigner: OptionalSigV4aSigner | null = null;
 
-      try {
-        CrtSignerV4 = signatureV4CrtContainer.CrtSignerV4;
-        if (typeof CrtSignerV4 !== "function") throw new Error();
-      } catch (e) {
-        e.message =
-          `${e.message}\n` +
-          `Please check whether you have installed the "@aws-sdk/signature-v4-crt" package explicitly. \n` +
-          `You must also register the package by calling [require("@aws-sdk/signature-v4-crt");] ` +
-          `or an ESM equivalent such as [import "@aws-sdk/signature-v4-crt";]. \n` +
-          "For more information please go to " +
-          "https://github.com/aws/aws-sdk-js-v3#functionality-requiring-aws-common-runtime-crt";
-        throw e;
-      }
+      if (signatureV4CrtContainer.CrtSignerV4) {
+        try {
+          CrtSignerV4 = signatureV4CrtContainer.CrtSignerV4;
+          if (typeof CrtSignerV4 !== "function") throw new Error();
+        } catch (e) {
+          e.message =
+            `${e.message}\n` +
+            `Please check whether you have installed the "@aws-sdk/signature-v4-crt" package explicitly. \n` +
+            `You must also register the package by calling [require("@aws-sdk/signature-v4-crt");] ` +
+            `or an ESM equivalent such as [import "@aws-sdk/signature-v4-crt";]. \n` +
+            "For more information please go to " +
+            "https://github.com/aws/aws-sdk-js-v3#functionality-requiring-aws-common-runtime-crt";
+          throw e;
+        }
+
+        this.sigv4aSigner = new CrtSignerV4({
+          ...this.signerOptions,
+          signingAlgorithm: 1,
+        });
+      } else if (signatureV4aContainer.SignatureV4a) {
+        try {
+          JsSigV4aSigner = signatureV4aContainer.SignatureV4a;
+          if (typeof JsSigV4aSigner !== "function") throw new Error();
+        } catch (e) {
+          e.message =
+            `${e.message}\n` +
+            `Please check whether you have installed the "@smithy/signature-v4a" package explicitly. \n` +
+            `You must also register the package by calling [require("@smithy/signature-v4a");] ` +
+            `or an ESM equivalent such as [import "@smithy/signature-v4a";]. \n`;
+          throw e;
```

Comment:
there should just be a single version of this message if neither are loaded. It should preserve the detailed instructions about how to statically load (one of) the packages and the link to our readme.

In the readme there should be an update for info on how to load a sigv4a impl and how to choose one.